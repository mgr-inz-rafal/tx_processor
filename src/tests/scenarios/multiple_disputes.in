type,client,tx,amount
deposit,1,1,50.0
deposit,1,2,25.0
dispute,1,1,
deposit,1,3,100.0
dispute,1,2,
deposit,1,4,1000.0
