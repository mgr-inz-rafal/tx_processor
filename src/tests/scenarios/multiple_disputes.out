client,available,held,total,locked
1,1100,75,1175,false